lib/rules/no-reservered-keys.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @fileoverview Prevent overwrite reserved keys
+ * @author Armano
+ */
+'use strict'
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+const RESERVER_NAMES = new Set(require('../utils/vue-reserved.json'))
+const SCOPE_NAMES = new Set(['props', 'computed', 'data', 'methods'])
+
+function create (context) {
+  const reservedNames = RESERVER_NAMES
+  const scopeNames = SCOPE_NAMES
+
+  const options = context.options[0] || {}
+  if (options.reserved) {
```

Comment:
You can actually simplify config with:
```
const options = context.options[0] || {}
const reservedKeys = new Set([...RESERVED_KEYS, ...options.reserved || []])
const scopes = new Set([...SCOPE_NAMES, ...options.scopes || []])
```
Plus you can then set `RESERVED_KEYS` and `SCOPE_NAMES` as arrays above.
I changed names of variables so keep it in mind :)